std/string.d

```diff
@@ -6046,7 +6095,6 @@ bool isNumeric(S)(S s, bool bAllowSep = false) if (isSomeString!S)
     assert(!isNumeric("+"));
 }
 
```

Comment:
Maybe a ddoc unittest with a bunch of `static` assertions + ddoc note about CTFE-ability.